## E1603 (unpacking-in-except)

### :x: Problematic code:

```python
# to be addded
```

### :heavy_check_mark: Correct code:

```python
# to be added
```

### Rationale:

Python3 will not allow implicit unpacking of exceptions in except clauses.
See http://www.python.org/dev/peps/pep-3110/

### Related resources:

- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22unpacking-in-except%22+OR+%22E1603%22)
